packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -190,6 +225,9 @@ class _ReorderableListContent extends StatefulWidget {
   final EdgeInsets padding;
   final ReorderCallback onReorder;
   final bool reverse;
+  final ScrollPhysics physics;
+  final bool shrinkWrap;
+  final bool primary;
```

Comment:
As far as I can tell, these properties aren't being used by the widget. Did you need to use them somewhere?

Reply:
I originally added them after looking at an issue that asked for `shrinkWrap` and `primary` to be added to `ReorderableListView`, but they're not being used, so they're not needed currently.